pages/doc/integrations_tas_howto.md

```diff
@@ -73,7 +73,7 @@ We've streamlined the getting started process so it involves a few simple steps
 
 1. Download the Tanzu Observability by Wavefront nozzle file from the [Tanzu Network](https://network.pivotal.io/)
 2. In Tanzu Ops Manager, install, configure, and deploy the nozzle. At a minimum:
-    1. In the **Wavefront Proxy Config** section, specify the Operations for Applications instance, an API token, and a user-friendly host name. <!--The API token that you provide and how it is generated depends on whether your Operations for Applications service is onboarded to VMware Cloud services. For details on API tokens, see [Managing API Tokens](wavefront_api.html#managing-api-tokens).-->
+    1. In the **Wavefront Proxy Config** section, specify the Operations for Applications instance, authentication credentials, and a user-friendly host name.
```

Comment:
thought: keep it simple in this overview section - mention credentials, maybe link to the more detailed explanation further down.